spyder/widgets/collectionseditor.py

```diff
@@ -619,14 +621,17 @@ def setup_table(self):
         # Sorting columns
         self.setSortingEnabled(True)
         self.sortByColumn(0, Qt.AscendingOrder)
+        self.selectionModel().selectionChanged.connect(self.refresh_menu)
 
     def setup_menu(self):
         """Setup context menu"""
         resize_action = create_action(self, _("Resize rows to contents"),
+                                      icon=ima.icon('collaprow'),
                                       triggered=self.resizeRowsToContents)
         resize_columns_action = create_action(
             self,
             _("Resize columns to contents"),
+            icon=ima.icon('collapcol'),

```

Comment:
And here too.

Reply:
Thank you @ccordoba12 I really forgot of these changes! Done!